pkg/transform/service.go

```diff
@@ -0,0 +1,11 @@
+package transform
+
+import (
+	"go.expect.digital/translate/pkg/model"
+	translatev1 "go.expect.digital/translate/pkg/pb/translate/v1"
+)
+
+// ProtoServiceFromService converts model.Service to translatev1.Service.
+func ProtoServiceFromService(service *model.Service) *translatev1.Service {
```

Comment:
Keep the naming simple. E.g.

```suggestion
func ServiceFromPb(service *translatev1.Service) *model.Service
func ServiceToPb(service *model.Service) *translatev1.Service

// or

func ServiceFromProto(service *translatev1.Service) *model.Service
func ServiceToProto(service *model.Service) *translatev1.Service
```